python-threatexchange/threatexchange/cli/dataset/simple_serialization.py

```diff
@@ -7,8 +7,10 @@
 import re
 import typing as t
 
-from threatexchange.fb_threatexchange import threat_updates
-from threatexchange.fb_threatexchange.descriptor import SimpleDescriptorRollup
+from threatexchange.exchanges.clients.fb_threatexchange import threat_updates
+from threatexchange.exchanges.clients.fb_threatexchange.descriptor import (
```

Comment:
totally-non-blocking:

Does it smell weird to you that a common serialization class depends on exchanges.clients.fb_threatexchange ?

Is this a candidate for externalizing down the road? Perhaps fb_threatexchange could expose serialization / deserialization methods?

Reply:
Yea, definitely does. Let me trace down what happened.